<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>311</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Anki QuickAdd</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="widget_layout">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" name="combobox_layout">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="deck_combobox_label">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Deck:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="card_combobox_label">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Card</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="deck_comboBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="card_comboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="form_label_layout">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="select_a_deck_label">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Select a card to enter input in fields&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textbox_1_label">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Card field 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textbox_1_lineedit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>example input for field 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textbox_2_label">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Card field 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textbox_2_lineedit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>more example input for field 2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="tags_layout">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="tags_label">
         <property name="text">
          <string>Tags:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="add_button">
       <property name="text">
        <string>Add to Anki</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
